lib/Migration/Version3003Date20180720162342.php

```diff
@@ -39,12 +39,12 @@ public function changeSchema(IOutput $output, \Closure $schemaClosure, array $op
 
 		if (!$table->hasColumn('object_type')) {
 			$table->addColumn('object_type', Type::STRING, [
-				'notnull' => true,
+				'notnull' => false,
```

Comment:
Why this change? Were not empty strings allowed already (in fact the default value is an empty string)?